### DISCUSSION TITLE
NBT Support

### DISCUSSION THREAD

**Discussion by Dalethium** [Zrips/CMI#3335] [Feature Requests]
It would be amazing if the mirror command and /worth command supported nbt and block data, for example, player skulls are set to steve heads in both of those.

**Comment by Dalethium** [comment]
Also another thing that I'd love to support it would be custom recipe inputs.

**Comment by mrfloris** [comment]
Custom items in setworth support is also requested here https://github.com/Zrips/CMI/issues/1572

**Comment by ghost** [comment]
Custom items with Modsupport  would be nice
